staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/generic/webhook.go

```diff
@@ -207,6 +222,43 @@ func (a *Webhook) ShouldCallHook(h webhook.WebhookAccessor, attr admission.Attri
 		return nil, matchObjErr
 	}
 
+	versionedAttr, err := admission.NewVersionedAttributes(attr, invocation.Kind, o)
```

Comment:
this will become a call to the interface, and can move inside the `if len(matchConditions) > 0 {` block

Reply:
done